src/components/Navigation.vue

```diff
@@ -0,0 +1,51 @@
+<template>
+  <nav class='nav'>
+    <div class='nav__items nav__items--left'>
+
+    </div>
+    <div class='nav__items nav__items--center'>
+      <span v-for='link in links' :key='link.url'>
+        <a :href='link.url'>{{ link.text }}</a>
+      </span>
+    </div>
+    <div class='nav__items nav__items--right'>
+      
+    </div>
+  </nav>
+</template>
+
+<script>
+  export default {
+    name: 'Navigation',
+    data: () => ({
+      links: [
+        { text: 'Home', url: '/' }
+      ]
+    })
+  }
+</script>
+
+
+<style lang='scss' scoped>
+  @import 'src/styles/mixins';
+  @import 'src/styles/variables';
+
+  .nav {
+    &__items {
```

Comment:
what do the `&__` and `&--` sass selectors do?

Reply:
Nothing, its just a methodology
http://getbem.com/introduction/

Reply:
yeah, i've used `&` but the prefixes are definitely something i'll have to get used to...

Reply:
Ya, luckily this is one of the easier methodologies to learn :p (compared to something like ATOMIC)
We could come up with another naming scheme or something if u want